crates/pessimistic-proof/src/global_index.rs

```diff
@@ -15,6 +17,10 @@ pub struct GlobalIndex {
 
 impl GlobalIndex {
     const MAINNET_FLAG_OFFSET: usize = 2 * 32;
+
+    pub fn network_id(&self) -> NetworkId {
```

Comment:
Are we sure we want to allow `NetworkId(0)` ?
Does `NetworkId(0)` useful for any action?

Reply:
Yeah `NetworkId(0)` corresponds to the L1 network